aspnetcore/web-api/jsonpatch.md

```diff
@@ -14,119 +14,54 @@ uid: web-api/jsonpatch
 
 This article explains how to handle JSON Patch requests in an ASP.NET Core web API.
 
-## Package installation
+JSON Patch support in ASP.NET Core web API is based on <xref:System.Text.Json> serialization, and requires the [`Microsoft.AspNetCore.JsonPatch.SystemTextJson`](https://www.nuget.org/packages/Microsoft.AspNetCore.JsonPatch.SystemTextJson) NuGet package.
 
-JSON Patch support in ASP.NET Core web API is based on `Newtonsoft.Json` and requires the [`Microsoft.AspNetCore.Mvc.NewtonsoftJson`](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson/) NuGet package. To enable JSON Patch support:
+**[JSON Patch](https://jsonpatch.com/)**:
```

Comment:
What is jsonpatch.com?

Reply:
jsonpatch.com is a website dedicated to the JSON Patch standard.  It is not helpful for me to just put the link in the name here though.  I will move the link out, put it in the description below and indicate to go there for an overview of JSON Patch and the RFC 6902 standard so it has proper context.  Thanks.

Reply:
Resolved.  Moved it and added context.